tests/linearGaussian_vector_field_test.py

```diff
@@ -205,18 +194,19 @@ def simulator(theta):
     )
 
 
-# TODO: This should be unified with NPSE when the network builders are unified
-# in PR #1501
-@pytest.mark.parametrize("model", ["mlp", "resnet"])
-def test_fmpe_with_different_models(model):
+@pytest.mark.parametrize("vector_field_type", [NPSE, FMPE])
+@pytest.mark.parametrize(
+    "model", ["mlp", "ada_mlp", pytest.param("transformer", marks=[pytest.mark.slow])]
+)
+def test_fmpe_with_different_models(vector_field_type, model):
     """Test fmpe with different vector field estimators on linear Gaussian."""
 
     theta_dim = 3
     x_dim = 3
 
     x_o = zeros(1, x_dim)
     num_samples = 1000
-    num_simulations = 2000
+    num_simulations = 2500
```

Comment:
Is this for the same reason above, that fmpe struggles a bit and so we give it more simulations?